### ISSUE TITLE
cnspec reports different User-Agents depending on destination

### ISSUE THREAD

**Issue by philipbalinov** [mondoohq/cnspec#1451]
**Describe the bug**
A single run of `cnspec scan local` reports multiple User-Agent strings when performing http(s) requests, seemingly based on the URL being called.

**To Reproduce**
Observed while running `cnspec scan local --api-proxy http://localhost:8080` against MITMProxy

**Expected behavior**
cnspec should report a consistent User-Agent string for all outgoing HTTP requests, for example in the format 
> User-Agent: PN/&lt;platform&gt; build/&lt;build shorthash&gt; cnspec/&lt;version&gt; cnquery/&lt;version&gt;

**Screenshots or CLI Output**
I can provide a full flow log from MITMProxy upon request. Here is the relevant information w.r.t. request path and reported user-agent 

TLDR; there are three User-agent strings here: `Go-http-client/1.1,`, `PN/linux build/4dd693e cnquery/11.24.0`, and `mondoo-graphql-client/0.0.1`

```
root@ubuntu:~# grep -aioP  '((path;)(.*?)),|((user-agent,)(.*?),)' cnspec-local-scan-flow.log
path;22:/providers/latest.json,
User-Agent,18:Go-http-client/1.1,
path;20:/PolicyHub/GetBundle,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;33:/PolicyResolver/SynchronizeAssets,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;27:/PolicyHub/GetPolicyFilters,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;36:/PolicyResolver/ResolveAndUpdateJobs,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;28:/PolicyResolver/StoreResults,
User-Agent,38:PN/linux build/4dd693e cnquery/11.24.0,
path;6:/query,
User-Agent,27:mondoo-graphql-client/0.0.1,
```

**Desktop (please complete the following information):**

- OS: Ubuntu
- OS Version: 22.04
- Browser if applicable: N/A
- Browser Version: N/A

**Additional context**
--
